diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
index a154b12d..0a9e4f06 100644
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -294,10 +294,10 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         </then>
         <else>
           <message name="IDS_PRODUCT_NAME" desc="The Chrome application name" translateable="false">
-            Chromium
+            AI-FREE
           </message>
           <message name="IDS_SHORT_PRODUCT_NAME" desc="The Chrome application short name." translateable="false">
-            Chromium
+            AI-FREE
           </message>
         </else>
       </if>
@@ -308,11 +308,11 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         </message>
         <message name="IDS_SHORTCUT_NAME_DEV" desc="Unused in Chromium builds" translateable="false">
         </message>
-        <message name="IDS_PRODUCT_DESCRIPTION" desc="Browser description">
-          Chromium is a web browser that runs webpages and applications with lightning speed. It's fast, stable, and easy to use. Browse the web more safely with malware and phishing protection built into Chromium.
+        <message name="IDS_PRODUCT_DESCRIPTION" desc="Browser description" translateable="false">
+          AI-FREE is a web browser that runs webpages and applications with speed, stability, and built-in protection against malicious sites.
         </message>
-        <message name="IDS_WELCOME_TO_CHROME" desc="Welcoming text announced via screen readers the first time Chrome is launched at the conclusion of installation.">
-          Welcome to Chromium; new browser window opened
+        <message name="IDS_WELCOME_TO_CHROME" desc="Welcoming text announced via screen readers the first time Chrome is launched at the conclusion of installation." translateable="false">
+          Welcome to AI-FREE; new browser window opened
         </message>
       </if>
       <if expr="is_macosx or is_linux">
@@ -354,8 +354,8 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         </message>
       </if>
       <if expr="not is_macosx and not is_chromeos">
-        <message name="IDS_TASK_MANAGER_TITLE" desc="The title of the Task Manager window">
-          Task Manager - Chromium
+        <message name="IDS_TASK_MANAGER_TITLE" desc="The title of the Task Manager window" translateable="false">
+          Task Manager - AI-FREE
         </message>
       </if>
       <message name="IDS_TASK_MANAGER_SEARCH_PLACEHOLDER" desc="The placeholder text for the Search bar in the Task Manager window">
@@ -388,8 +388,8 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
       <message name="IDS_TASK_MANAGER_CATEGORY_ALL_NAME" desc="The title of the 'All tasks' filter category on Windows/Mac/Linux/ChromeOS.">
         All tasks
       </message>
-      <message name="IDS_SESSION_CRASHED_VIEW_UMA_OPTIN" desc="Text besides the checkbox to let users optin to UMA in the restore from previous crash bubble.">
-        Help make Chromium better by sending crash reports and <ph name="UMA_LINK">$1<ex>usage statistics</ex></ph> to Google
+      <message name="IDS_SESSION_CRASHED_VIEW_UMA_OPTIN" desc="Text besides the checkbox to let users optin to UMA in the restore from previous crash bubble." translateable="false">
+        Help make AI-FREE better by sending crash reports and <ph name="UMA_LINK">$1<ex>usage statistics</ex></ph>
       </message>
       <if expr="not is_android">
         <message name="IDS_CHROME_SIGNOUT_CONFIRMATION_PROMPT_TOO_MANY_BOOKMARKS_BODY" desc="Body of the signout confirmation dialog when the user has too many bookmarks.">
@@ -405,10 +405,10 @@ New bookmarks can’t be saved in your account until you delete some. If you sig
 New bookmarks can’t be saved in your account until you delete some. If you sign out now, unsaved bookmarks will be deleted.
         </message>
       </if>
-      <message name="IDS_RESET_PROFILE_SETTINGS_EXPLANATION_IN_BULLET_POINTS" desc="A label formatted with bullet points describing the consequences of the 'reset profile settings' feature">
+      <message name="IDS_RESET_PROFILE_SETTINGS_EXPLANATION_IN_BULLET_POINTS" desc="A label formatted with bullet points describing the consequences of the 'reset profile settings' feature" translateable="false">
         This action will:
         <ph name="LINE_BREAKS">&lt;br&gt;&lt;br&gt;</ph>
-          • Reset some Chromium settings and Chromium shortcuts
+          • Reset some AI-FREE settings and shortcuts
           <ph name="LINE_BREAK">&lt;br&gt;</ph>
           • Disable extensions
           <ph name="LINE_BREAK">&lt;br&gt;</ph>
@@ -425,8 +425,8 @@ New bookmarks can’t be saved in your account until you delete some. If you sig
             </message>
           </then>
           <else>
-            <message name="IDS_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for titles displayed in tabs and popup windows">
-              <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - Chromium
+            <message name="IDS_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for titles displayed in tabs and popup windows" translateable="false">
+              <ph name="PAGE_TITLE">$1<ex>Example</ex></ph> - AI-FREE
             </message>
           </else>
         </if>
@@ -439,8 +439,8 @@ New bookmarks can’t be saved in your account until you delete some. If you sig
       </if>
       <if expr="not is_chromeos and not is_macosx">
         <!-- Captive Portal Browser Window Title Format -->
-        <message name="IDS_CAPTIVE_PORTAL_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for titles displayed in captive portal popup windows">
-          <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - Network Sign-in - Chromium
+        <message name="IDS_CAPTIVE_PORTAL_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for titles displayed in captive portal popup windows" translateable="false">
+          <ph name="PAGE_TITLE">$1<ex>Example</ex></ph> - Network Sign-in - AI-FREE
         </message>
       </if>
       <if expr="is_chromeos">
@@ -464,8 +464,8 @@ New bookmarks can’t be saved in your account until you delete some. If you sig
           </message>
         </then>
         <else>
-          <message name="IDS_ACCESSIBLE_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for the accessible name of a tabbed browser window">
-            <ph name="PAGE_TITLE">$1<ex>Google</ex></ph> - Chromium
+          <message name="IDS_ACCESSIBLE_BROWSER_WINDOW_TITLE_FORMAT" desc="The format for the accessible name of a tabbed browser window" translateable="false">
+            <ph name="PAGE_TITLE">$1<ex>Example</ex></ph> - AI-FREE
           </message>
         </else>
       </if>
